apps/dotcom/sync-worker/src/types.ts

```diff
@@ -69,6 +69,11 @@ export function isDebugLogging(env: Environment) {
 	return env.TLDRAW_ENV === 'development' || env.TLDRAW_ENV === 'preview'
 }
 
+export function getUserDoSnapshotKey(env: Environment, userId: string) {
+	const snapshotPrefix = env.TLDRAW_ENV === 'preview' ? env.WORKER_NAME + '/' : ''
```

Comment:
Should we also add some prefix for staging and production? Collision chance is very low, but might help if we ever want to clean just one of those but not the other.

Reply:
staging and production have their own individual R2 buckets so there's no chance of leaking between them

Reply:
Ah right.